src/libraries/System.Threading.Tasks.Dataflow/src/System.Threading.Tasks.Dataflow.csproj

```
@@ -60,7 +61,16 @@
   <ItemGroup>
     <Content Include="XmlDocs\System.Threading.Tasks.Dataflow.xml" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetFramework)' != 'netstandard2.0'">
+  <ItemGroup Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETCoreApp'">
+    <Reference Include="System.Collections" />
+    <Reference Include="System.Collections.Concurrent" />
+    <Reference Include="System.Linq" />
+    <Reference Include="System.Runtime" />
+    <Reference Include="System.Threading" />
+    <Reference Include="System.Threading.Thread" />
+    <Reference Include="System.Threading.ThreadPool" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard1.0' or '$(TargetFramework)' == 'netstandard1.1' or $(TargetFramework.StartsWith('net4'))">
```

nit, if the tfm isn't rid specific, just use equality checks like for netstandard.

```suggestion
  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard1.0' or '$(TargetFramework)' == 'netstandard1.1' or '$(TargetFramework)' == 'net461'">
```

---

I copied that from the top of this file:

```xml
<DefineConstants Condition="'$(TargetFramework)' == 'netstandard1.1' or '$(TargetFramework)' == 'netstandard2.0' or $(TargetFramework.StartsWith('net4'))">$(DefineConstants);FEATURE_TRACING</DefineConstants>
```

Should I fix that one as well, if that's how the condition should be written?

---

I'm fine with either. I will go over the libraries projects at some point anyway and clean-up the conditions. The new msbuild property functions that shipped with NET5 allow to condition on the TargetFrameworkIdentifier, TargetFrameworkPlatform, etc. directly in the project file which will simplify most of our conditions.